nexus/db-queries/src/db/datastore/device_auth.rs

```diff
@@ -176,4 +181,64 @@ impl DataStore {
                 )
             })
     }
+
+    // Similar to session hard delete and silo group list, we do not do a
+    // typical authz check, instead effectively encoding the policy here that
+    // any user is allowed to list and delete their own tokens. When we add the
+    // ability for silo admins to list and delete tokens from any user, we will
+    // have to model these permissions properly in the polar policy.
```

Comment:
This is important. We hack around the harder authz problem of modeling token properly by restricting these functions to the current actor (for now).